### PR TITLE
Add Scope token mints

### PR DESCRIPTION
**Pull request by peroxy**
* Refactor constants into separate files
* Export everything
* Remove SaberMSOL
* Add scope tests to CI

### CODE REVIEW

File: packages/scope-sdk/src/constants/Mints.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  { token: 'USDR', mint: 'USDrbBQwQbQ2oWHUPfA8QBHcyVxKUq1xHyXsSLKdUq2' },
+  { token: 'RATIO', mint: 'ratioMVg27rSZbSvBopUvsdrGUzeALUfFma61mpxc8J' },
+  { token: 'UXP', mint: 'UXPhBoR3qG4UCiGNJfV7MqhHyFqKN68g45GoYvAeL2M' },
+  { token: 'kUXDUSDCOrca', mint: '4G9USgnbg6fDTQ5AUfpCjM89zqbzWj32xfqvsaAu66DM' },
+  { token: 'JITOSOL', mint: 'J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn' },
+  { token: 'BONK', mint: 'DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263' },
+  { token: 'SAMO', mint: '7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU' },
+  { token: 'bSOL', mint: 'bSo13r4TkiE4KumL71LsHTPpL2euBYLFx6h9HP3piy1' },
+  { token: 'LaineSOL', mint: 'LAinEtNLgpmCP9Rvsf5Hn8W6EhNiKLZQti1xfWMLy6X' },
+  { token: 'HADES', mint: 'BWXrrYFhT7bMHmNBFoQFWdsSgA3yXoAnMhDK6Fn1eSEn' },
```
Comment (silviutroscot) [inline packages/scope-sdk/src/constants/Mints.ts:55]: please update the docs for creating new strats and adding new tokens to keep this one up to date as well

File: packages/scope-sdk/tests/sdk.test.ts [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+import { expect } from 'chai';
 
 describe('Scope SDK Tests', () => {
-  let connection: Connection;
-  const cluster = 'mainnet-beta';
-
-  beforeAll(() => {
-    connection = new Connection(clusterApiUrl(cluster));
-  });
+  // let connection: Connection;
```
Comment (silviutroscot) [inline packages/scope-sdk/tests/sdk.test.ts:6]: do you need this or can be removed?

Comment (peroxy) [reply packages/scope-sdk/tests/sdk.test.ts:6]: Not really needed for unit tests, will remove

File: packages/scope-sdk/src/constants/Mints.ts (hunk 1 above)
Comment (peroxy) [reply packages/scope-sdk/src/constants/Mints.ts:55]: Will update and add to one of my PRs that I have in historian, thanks for the reminder

**Review by silviutroscot — APPROVED** [review]

**Pull request merged**